backend/events/kotaeexpo2024/management/commands/setup_kotaeexpo2024.py

```diff
@@ -292,6 +295,116 @@ def setup_directory(self):
             active_until=self.event.end_time + timedelta(days=30),
         )
 
+    def setup_forms(self):
+        from forms.models.dimension import DimensionDTO
+        from forms.models.form import Form
+        from forms.models.survey import Survey
+
+        # Dance judge signup form
+
+        dance_judge_signup_survey, _ = Survey.objects.get_or_create(
+            event=self.event,
+            slug="dance-judge-signup",
+            defaults=dict(
+                active_from=now(),
+                key_fields=["name"],
+            ),
+        )
+
+        with resource_stream("events.kotaeexpo2024", "forms/dance-judge-signup-dimensions.yml") as f:
+            data = yaml.safe_load(f)
+
+        for dimension in data:
+            DimensionDTO.model_validate(dimension).save(dance_judge_signup_survey)
+
+        with resource_stream("events.kotaeexpo2024", "forms/dance-judge-signup-fi.yml") as f:
+            data = yaml.safe_load(f)
+
+        dance_judge_signup_fi, created = Form.objects.get_or_create(
+            event=self.event,
+            slug="dance-judge-signup-fi",
+            language="fi",
+            defaults=data,
+        )
+
+        # TODO temporary for development
```

Comment:
?

Reply:
when there eventually will be a form editor, the idea is that one may initialize a form from a template that is stored as a yaml file in the repository, but they would not like changes made in the form editor to be overwritten by whatever comes from the repository

on the other hand, now that we do not yet have the form editor mounted & brought to 2024, the primary way to update the forms is by editing yaml files in the repository. hence we want repository changes to override what is in the database

once the form editor is mounted, we will probably want to remove these blocks here, in `setup_hitpoint2024.py` and in `setup_tracon2024.py` and let die turisten und nonteknischen lookenpeepers mit seinem cottonpicker händer gewerken bei dummkopfen